provider/schema/execution_test.go

```diff
@@ -316,6 +316,26 @@ func TestExecutionData_ResolveTable(t *testing.T) {
 		assert.Equal(t, []string{"name", "name_no_prefix", "prefix_name", "cq_id", "meta", "injected_field"}, expectedResource.columns)
 		assert.Equal(t, []interface{}{"other", "name_no_prefix", "prefix_name", expectedResource.cqId, expectedResource.Get("meta"), 1}, values)
 	})
+
+	t.Run("test partial fetch", func(t *testing.T) {
```

Comment:
I think this is a great test but move moving parts can fail in a fetch as stated above